libs/blocks/icon-block/icon-block.js

```diff
@@ -14,44 +14,49 @@
 * Icon Block - v5.1
 */
 
-import { decorateBlockBg, decorateButtons } from '../../utils/decorate.js';
-import { createTag } from '../../utils/utils.js';
+import { decorateButtons } from '../../utils/decorate.js';
 
-function decorateLayout(el) {
-  const children = el.querySelectorAll(':scope > div');
-  if (children.length > 1 && children[0].childNodes.length) {
-    decorateBlockBg(el, children[0]);
-  }
-  const foreground = createTag('div', { class: 'foreground' });
-  el.appendChild(foreground);
-  return foreground;
+const iconBlockVariants = {
+  fullwidth: ['XL', 'M'],
+  small: ['L', 'M'],
+  bio: ['S', 'S'],
+  vertical: ['S', 'M'],
+  centered: ['M', 'M']
 }
```

Comment:
Do you have the eslint plugin? These should get flagged by that.
```suggestion
  centered: ['M', 'M'],
};
```

Reply:
Looks like it's still missing the semicolon

Reply:
awe good eye 👁️ added it.. thanks!